test/README.ssltest.md

```diff
@@ -89,6 +89,9 @@ handshake.
 
 * ExpectedTmpKeyType - the expected algorithm or curve of server temp key
 
+* ExpectedServerKeyType, ExpectedClientKeyType - the expected algorithm or
```

Comment:
No and we don't really support client certificate selection. I could add a minimal case to just test the client cert type is RSA in the relevant tests though.